packages/generator-converters/src/wirePropertyV2ToSdkPrimaryKeyTypeDefinition.ts

```diff
@@ -46,6 +46,8 @@ export function wirePropertyV2ToSdkPrimaryKeyTypeDefinition(
     case "float":
     case "geotimeSeriesReference":
     case "mediaReference":
+    case "struct":
```

Comment:
I know we want to ignore these property types instead of throwing errors for unsupported, but saving as a separate PR since structs is higher prio.